src/Backend/Core/Layout/Templates/macros.html.twig

```diff
@@ -8,3 +8,23 @@
 {% macro required() %}
   <abbr data-toggle="tooltip" title="{{ 'lbl.RequiredField'|trans|ucfirst }}">*</abbr>
 {% endmacro %}
+
+{% macro dataGrid(dataGrid, noItemsMessage) %}
+  <div class="row">
+    <div class="col-md-12">
```

Comment:
nope, when used in an empty content block, the structure will be `.container>.row>.col-md-12` which is bootstrap's standard way of working. If you use this macro in some other markup   leaving the row and col out might cause some layout issues